Visual Studio Code docs (#263)
sb2nov/mac-setup: VisualStudioCode/README.md

# Visual Studio Code

[Visual Studio Code](https://code.visualstudio.com/) is a lightweight code editor with support for many programming langagues through [extensions](https://code.visualstudio.com/docs/editor/extension-gallery)

## Installation

> To install the latest version, use homebrew:

Reviewer: ```suggestion
To install the latest version, use Homebrew:
```

```bash
brew cask install visual-studio-code
```

> ## MacOS integration

Reviewer: ```suggestion
## macOS integration
```

> Launch VSCode from [command line](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line)

Reviewer: ```suggestion
Launch VS Code from the [command line](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line).
```

> After that, you can launch VSCode from your terminal :

Reviewer: ```suggestion
After that, you can launch VS Code from your terminal:
```

> * `code .` will open VSCode in the current directory

Reviewer: ```suggestion
* `code .` will open VS Code in the current directory
```

> * `code myfile.txt` will open `myfile.txt` in VSCode

Reviewer: ```suggestion
* `code myfile.txt` will open `myfile.txt` in VS Code
```
